agent-scala/src/main/scala/kamon/agent/scala/KamonInstrumentation.scala

```diff
@@ -50,6 +52,10 @@ trait KamonInstrumentation extends kamon.agent.api.instrumentation.KamonInstrume
     super.forTargetType(name, builder)
   }
 
+  def forType(typeDescription: => ElementMatcher[_ >: TypeDescription])(builder: InstrumentationDescription.Builder ⇒ InstrumentationDescription): Unit = {
```

Comment:
@dpsoft `forType()` receives an ElementMatcher so you can make a super custom ElementMatcher from the module you are instrumenting and introduce in the KamonInstrumentation. I thought that I was going to need it while I was instrumenting the next modules, but it was not the case haha